### ISSUE TITLE
Bug: Extractor archive.org "Failed to find "content-location" URL header in Archive.org response."

### ISSUE THREAD

**Issue by robertospenha** [ArchiveBox/ArchiveBox#1570]
Hello developers

I'm using the stable docker image called latest on dockerhub from 6 months ago.

I'm running on an ubuntu 24.04 virtual machine with gnome in a situation where I have almost no additional programs other than the standard ubuntu and I've created other VMs for testing just for that.

In all situations I had the same error even using different sites to save the sites.

Two items always give an error and don't save, which are “git” and “archive.org”. as shown in the image below they are transparent.

What bothered me most was that the downloaded site was not passed on to archive.org.

![Image](https://github.com/user-attachments/assets/44e93ea2-acd2-47e4-bf95-107d2f6a77bb)

![Image](https://github.com/user-attachments/assets/32d983ab-f479-415c-b87d-adefa01e2b66)

Another thing that made me realize that it wasn't just me yesterday and today. It was a recent video on youtube in which the channel presents the archivebox project and used docker compose and had the same problem on several sites even though he didn't mention this apparent limitation

![Image](https://github.com/user-attachments/assets/4ff8bdfc-a568-48b3-a89c-882f7af651be)

Link video: https://www.youtube.com/watch?v=jNeybjyxtMM

Thank you for your attention and I look forward to hearing from you

**Comment by robertospenha** [comment]
I think you might have a reason about the archive.org..

It may have been related to the hacker attack they had.

Second post on their blog is temporarily suspended uploading to the site.

https://blog.archive.org/2024/10/21/internet-archive-services-update-2024-10-21/

How I started using the archivebox this week could have been exactly why. And the case mentioned above the channel of youtube may have been because of this also since it is a recent video.

.

**Comment by pirate** [comment]
Yeah I was about to post the same, Archive.org has been in read-only mode the last week. It's expected for any new URLs to fail saving because they disabled Save-Page-Now during the attack.

**Comment by jessienab** [comment]
So this is still happening, well after the Archive.org attack... surely other add-ons that push to Save Page Now have been working.

Archivebox 0.7.3

**Comment by reliqwastaken** [comment]
i get this too

**Comment by pirate** [comment]
This is an upstream error not due to ArchiveBox, it can happen when archive.org:

- ratelimits saving a certain URL that has already been saved many times
- has downtime or degraded service on their savepagenow system (it's often the first system to be turned off when they're having infra problems)
- the domain owner has requested the URL be excluded from archive.org
- archiving takes too long and the request times out before archive.org is able to capture a copy (doesn't always mean it isn't saved, just means we didn't confirm the save succeeded in time).
- your IP has been individually ratelimited or banned due to excessive/abusive traffic
- your IP range has been blocked due to sharing an ASN or IP range with other abusive traffic (most commonly seen if you're archiving from a digitalocean/AWS/Hertzner/Azure/GCP/etc. IP)

For context I see Archive.org errors on about 10~20% of the URLs I save during normal usage, so there is a base level expected failure rate that's basically out of our control. Archivebox automatically retries saving anytime you run `archivebox update` so you can keep trying periodically until it saves if the failure is transient.

Common urls that fail to save are things like:

- popular domains like `example.com`, `google.com`, `reddit.com`, `news.ycombinator.com`, etc. that have already been saved by their automated crawling scripts / that they don't want to save 1000x per day just because 1000 people re-submit it
- private facebook.com/instagram/tiktok/google docs/etc. urls that they cant access without cookies or captcha-solving
- archive.org urls (they prevent recursive archiving of their own urls)

If you can share some details about specific URLs that are failing for you I can contact friends on the Archive.org team and investigate further.

**Comment by reliqwastaken** [comment]
ah ok. all good thanks, its fine if i just dont use archive.org
